wiki/entry_ehret.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "Wiki Entry Template"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Maren Ehret"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+  - "Digital Healthcare"
+  - Paper: Schmietow, B. & Marckmann, G. (2019). Mobile health ethics and the expanding role of autonomy. Medicine, health care, and philosophy, 22(4),               623–630. 
+---
+
+Key Quote: "Autonomous, empowered individuals taking care proactively of their health are pictured as the creators of this vision for future 
+healthcare, and may at the same time redefne their role in relation to the traditional, expert-driven healthcare system." (Schmietow & Marckmann, 2019)
+
+Main Argument: The article by Schmietow and Marckmann is part of a larger project in the debate about ethical issues of digital healthcare and focuses 
+on the role of autonomy regarding new digital health applications. With the growing of various mobile health (mhealth) technologies, a lot of ethical questions as well as questions about the user/patient's autonomy continue to arise. Schmietow and Marckmann argue that mhealth strengthens and empowers
+their independancy in making decisions about their own health. They continue to explain how more autonomy and self-determination can benefit and support
+self-care and finally create a form of democratised medecine. However, they do highlight the imortance of digital and health literacy.
+
```

Comment:
Do you think the aspects mentioned here can have an influence on individuals' autonomy need experiences? If yes, how so? If no, why not?

Reply:
Based on the definition of autonomy need experience by Vansteenkiste, I think that more independance concerning one's health decisions has an influence on their autonomy need experience. People start acting by themselves without having to follow direct directions by doctors in real life. They get to participate in their own treatment and can discuss it with health caretakers. This form of democratised medecine could certainly satisfy their volition and willingness in their actions. However if they fail or their knowledge ends at some point, they might in fact need speicif help from more experienced individuals and can therefore experience pressure and inner conflict which emerge from unsatisfied autonomy need experiences.